mobile/src/views/VerificationView.jsx

```diff
@@ -60,125 +71,162 @@ const VerifcationView = ({navigation}) => {
-            resizeMode="contain"></Image>
-        </View>
-        <View style={{alignItems: 'center', justifyContent: 'center'}}>
-          <View style={image ? styles.uploaded : styles.uploadSection}>
-            {image ? (
-              <FontAwesome5
-                name="file-alt"
-                size={64}
-                color={colors.primary}
-                backgroundColor="transparent"></FontAwesome5>
-            ) : (
-              <FontAwesome5
-                name="image"
-                size={64}
-                color={colors.primary}
-                backgroundColor="transparent"></FontAwesome5>
-            )}
-            {image ? (
-              <Text style={styles.filename}>{filename}</Text>
-            ) : (
-              <Pressable onPress={pickImage} style={styles.uploadButton}>
-                <Text style={styles.uploadButtonText}>Upload Photo</Text>
-              </Pressable>
-            )}
-            {image ? (
-              <View style={styles.cancelButton}>
-                <FontAwesome5.Button
-                  name="times"
-                  backgroundColor="transparent"
+          <Text style={styles.example}>Example</Text>
+          <View style={styles.exampleImage}>
+            <Image
+              source={require('../../assets/example.png')}
+              resizeMode="contain"></Image>
+          </View>
+          <View style={{alignItems: 'center', justifyContent: 'center'}}>
+            <View style={image ? styles.uploaded : styles.uploadSection}>
+              {image ? (
+                <FontAwesome5
+                  name="file-alt"
+                  size={64}
                   color={colors.primary}
-                  onPress={() => {
-                    setImage(null);
-                  }}
-                />
-              </View>
-            ) : (
-              <></>
-            )}
+                  backgroundColor="transparent"></FontAwesome5>
+              ) : (
+                <FontAwesome5
+                  name="image"
+                  size={64}
+                  color={colors.primary}
+                  backgroundColor="transparent"></FontAwesome5>
+              )}
+              {image ? (
+                <Text style={styles.filename}>{filename}</Text>
+              ) : (
+                <Pressable onPress={pickImage} style={styles.uploadButton}>
+                  <Text style={styles.uploadButtonText}>Upload Photo</Text>
+                </Pressable>
+              )}
+              {image ? (
+                <View style={styles.cancelButton}>
+                  <FontAwesome5.Button
+                    name="times"
+                    backgroundColor="transparent"
+                    color={colors.primary}
+                    onPress={() => {
+                      setImage(null);
+                    }}
+                  />
+                </View>
+              ) : (
+                <></>
```

Comment:
Optional nit: If we aren't rendering anything in the else block we can just `image && <YourComponent/>`

Reply:
latest commit refactored this conditional